app/controllers/admin/users_controller.rb

```diff
@@ -489,7 +489,10 @@ def total_other_accounts_with_same_ip
 
   def anonymize
     guardian.ensure_can_anonymize_user!(@user)
-    if user = UserAnonymizer.new(@user, current_user).make_anonymous
+    opts = {}
+    opts[:anonymize_ip] = params[:anonymize_ip] if params[:anonymize_ip].present?
```

Comment:
I wonder, what happens if the `anonymize_ip` is bogus?

Reply:
It seems to update to `nil` in that case. No failures or anything.